python/ingestion/gcs_to_bq_util.py

```diff
@@ -239,15 +239,26 @@ def load_csv_as_df_from_web(url, dtype=None, params=None, encoding=None):
     return pd.read_csv(url, dtype=dtype, encoding=encoding)
 
 
-def load_csv_as_df_from_data_dir(directory, filename, dtype=None):
+def load_csv_as_df_from_data_dir(directory=None, filename=None,
+                                 dtype=None, skiprows=None,
+                                 thousands=None, subdirectory=None):
     """Loads csv data from /data/{directory}/{filename} into a DataFrame.
        Expects the data to be in csv format, with the first row as the column
        names.
 
     directory: directory within data to load from
-    filename: file to load the csv file from"""
-    file_path = os.path.join(DATA_DIR, directory, filename)
-    return pd.read_csv(file_path, dtype=dtype)
+    filename: file to load the csv file from
+    subdirectory: combined directory and filename path
+    skiprows: how many rows to skip when reading csv
+    thousands: reads commas in the csv file as a thousand place indicator
+    """
+    if subdirectory is None:
+        file_path = os.path.join(DATA_DIR, directory, filename)
+
+    else:
+        file_path = os.path.join(DATA_DIR, subdirectory)
```

Comment:
i think `directory` and `filename` should always be required, and only `subdirectory` should be optional. then this line could be something like `file_path = os.path.join(DATA_DIR, directory, subdirectory, filename)`